### PR TITLE
SFDCENG-765 : Patch SFDC Canvas Framework SDK

### PR DESCRIPTION
**Pull request by CezarLeahu**
- update jackson
   - add missing fields

### CODE REVIEW

**Comment by montgolfiere** [comment]
Do we need to also

**Comment by montgolfiere** [comment]
Oops - trigger happy - mean't to hit "Comment" (not "Close & Comment") !

**Comment by montgolfiere** [comment]
Do we need to bump version to 1.2-minimium-SNAPSHOT or similar ?

File: src/main/java/canvas/CanvasContext.java [hunk 1]
```diff
@@ -120,15 +120,15 @@ public void setLinkContext(CanvasLinkContext linkContext)
...
+
     @Override
     public String toString()
     {
-        return String.format("Canvas Context:\n\t" + 
+        return String.format("Canvas Context:\n\t" +
                              "User Context:\n\t\t%s\n\t"+
                              "Org Context:\n\t\t%s\n\t"+
                              "Environment Context:\n\t\t%s\n\t"+
-                             "Link Context:\n\t\t%s\n",
```
Comment (montgolfiere) [inline src/main/java/canvas/CanvasContext.java]: Did anything change in this file other than formatting ? Remove from PR ?

Comment (CezarLeahu) [reply src/main/java/canvas/CanvasContext.java]: Done.

**Comment by CezarLeahu** [comment]
My IDE was automatically removing all the trailing spaces. I disabled that feature for now, and I removed the files that only had whitespace changes from the PR.

As for the version, I'm not sure what we should use. So far, the is nothing published with "**1.0.1-RELEASE-Minimum**" in [Nexus](https://artifacts.alfresco.com/nexus/content/groups/internal/sfdc/sfdc-canvas-sdk/), but I'm not sure what version string we should use.

**Comment by montgolfiere** [comment]
Yes:  adding "w=1" query param is an option.

**Comment by tzclucian** [comment]
GitHub power-user 😎

**Comment by CezarLeahu** [comment]
Can I merge this?

**Pull request merged**